docs/coding.rst

```diff
@@ -216,6 +264,38 @@ valid variation were given in the URL.
     a mixin that implements this pattern that should work with most views:
     `bedrock.utils.views.VariationMixin`.
 
+
+Geo Redirect View
+^^^^^^^^^^^^^^^^^
+
+We sometimes need to have a special page variation for people visiting from certain
+countries. To make this easier we have a redirect view class that will allow you to
+define URLs per country as well as a default for everyone else. This redirector URL
+must only be a redirector since it must be uncachable by our CDN so that all visitors
+will hit the server and see the correct page for their location.
+
+.. code-block:: python
+
+    from bedrock.base.views import GeoRedirectView
+
+    class CanadaIsSpecialView(GeoRedirectView):
+        geo_urls = {
+            'CA': 'app.canada-is-special',
+        }
+        default_url = 'app.everyone-else'
+
+In this example people in Canada would go to the URL that Django returns using `reverse()`
+(i.e. the name of the URL) and everyone else would go to the `app.everyone-else` URL. You
+may also use full URLs instead of URL names if you want to. It will look for strings that
+start with "http(s)://" and use it as is. The country code must be 2 characters and upper case.
```

Comment:
Is it worth linking `country code` here to a full list of valid codes?

Reply:
Good idea. I linked it to the official ISO 3166-1 alpha-2 list on wikipedia.